src/test/java/codewars/tasks/java/LeapYearsTest.java

```diff
@@ -0,0 +1,18 @@
+package codewars.tasks.java;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class LeapYearsTest {
+
+    @Test
```

Comment:
test name you can use in next way:
`@Test(testName = "testId",
        description = "Verify Leap Year method")`